### PR TITLE
renaming

### PR DESCRIPTION
**Pull request by ilanabonell**
- fix job
- Template - > sports-near-me/sportsnearme for func

### CODE REVIEW

File: server/src/env/env.go [hunk 1]
```diff
@@ -2,13 +2,14 @@ package env
...
 	"os"
+
+	"github.com/golang-jwt/jwt"
 )
 
 const (
-	jwtSecretEnvVarName = "TEMPLATE_JWT_SECRET"
-	apiTokenEnvVarName  = "TEMPLATE_API_TOKEN"
+	jwtSecretEnvVarName = "sports-near-me_JWT_SECRET"
+	apiTokenEnvVarName  = "sports-near-me_API_TOKEN"
```
Comment (edverma) [inline server/src/env/env.go:12]: All caps for environment variables

File: server/src/jobs/template_job.go [hunk 2]
```diff
@@ -3,30 +3,31 @@ package jobs
 import (
```
[the remark this answers is not in the document]
Comment (ilanabonell) [reply server/src/jobs/template_job.go:3]: Do you want me to delete dump.rdb file?

File: server/src/jobs/job.go [hunk 3]
```diff
@@ -18,7 +19,7 @@ type job struct {
 }
 
 func RunAllJobs(ctx context.Context) {
-	go RunTemplateJob(ctx)
+	go runSportsNearMeJob(ctx)
```
Comment (edverma) [inline server/src/jobs/job.go:22]: Capital (this makes the function exported)

File: server/src/jobs/sports-near-me_job.go [hunk 4]
```diff
@@ -3,30 +3,31 @@ package jobs
...
 )
 
-const logFileName = "template_job.log"
-const logPrefix = "template_job: "
+const logFileName = "sports-near-me_job.log"
+const logPrefix = "sports-near-me_job: "
 const pace = 1 * time.Second
 
-func RunTemplateJob(parentCtx context.Context) {
+func runSportsNearMeJob(parentCtx context.Context) {
```
Comment (edverma) [inline server/src/jobs/sports-near-me_job.go:15]: Capitalize

File: server/src/jobs/template_job.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (edverma) [reply server/src/jobs/template_job.go:3]: Yes

File: server/src/jobs/sports-near-me_job.go [hunk 5]
```diff
@@ -3,30 +3,31 @@ package jobs
...
 	if err != nil {
-		panic(fmt.Sprintf("failed to initialize template job. error: %v", err))
+		panic(fmt.Sprintf("failed to initialize sports-near-me job. error: %v", err))
 	}
 	s.StartBlocking()
 }
 
-func (j *job) templateJob(cron gocron.Job) {
-	j.l.Printf("running template job....")
+func (j *job) SportsNearMeJob(cron gocron.Job) {
```
Comment (edverma) [inline server/src/jobs/sports-near-me_job.go:29]: This should start with lowercase s. 

https://go.dev/tour/basics/3 for more info on exported/unexported functions.

**Review by edverma — APPROVED** [review]

**Pull request merged**